project/properties/views.py

```diff
@@ -11,6 +11,9 @@
 def show_all(request):
 	return render(request, 'properties/properties.html')
 
+def deliver_props(request):
```

Comment:
I called it deliver because it is the route the AJAX request will hit to receive the property delivery. It is the method that will grab and deliver the properties.